packages/api-v2/src/hooks/useOnfido.ts

```diff
@@ -7,6 +7,16 @@ import type { SdkHandle, SdkResponse } from '../types/onfido';
 import useOnfidoNotificationEvent from './useOnfidoNotificationEvent';
 import { v4 as uuidv4 } from 'uuid';
 
+// TODO: replace with the respective function from @deriv-com/utils
```

Comment:
why are we writing custom function?Can u specify whts the issue with @deriv-com/utils function

Reply:
I created a PR to deriv-utils, @amina-deriv 
https://github.com/deriv-com/deriv-utils/pull/74